templating/base_render_options.go

```diff
@@ -123,24 +123,80 @@ func OverrideValuesWithBuildInfo(c1 *Config, c2 *Config, opts *BaseRenderOptions
 	return base, nil
 }
 
-// LoadAndRenderSteps loads a template file and renders it according to an optional values file, --set values,
+// LoadAndRenderBuildSteps loads a template file for build and renders it according to an optional values file, --set values,
+// and base render options.
+func LoadAndRenderBuildSteps(ctx context.Context, template *Template, opts *BaseRenderOptions) (string, error) {
```

Comment:
Can you add unit tests for the new methods?

Reply:
@northtyphoon Sure, thanks